.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -3,7 +3,7 @@ set -exu
 
 ZENKO_NAME=${1:-end2end}
 COMMAND=${2:-"premerge"}
-PARALLEL_RUNS=${3:-2}
+PARALLEL_RUNS=${3:-6}
```

Comment:
can this be computed based on `nproc` ?

Reply:
Yes definitely, I will try today to optimize how many vCPUs CTST can use safely (I think nproc - 3 considering what was observed on 4core runners)